src/notificationStack.js

```diff
@@ -22,6 +24,7 @@ const NotificationStack = props => (
       const isLast = index === 0 && props.notifications.length === 1;
       const barStyle = props.barStyleFactory(index, notification.barStyle);
       const activeBarStyle = props.activeBarStyleFactory(index, notification.activeBarStyle);
+      const onClick = (notification.onClick || props.onClick) || defaultClick;
```

Comment:
Could you change this to just `notification.onClick || props.onClick`? Instead of using `defaultClick` you can use [a default prop value](https://github.com/pburtchaell/react-notification/blob/master/src/notificationStack.js#L49).